src/PackagePublisher/PackagePublisher.nuspec

```diff
@@ -8,5 +8,6 @@
   </metadata>
   <files>
     <file src="netcoreapp1.0\" target="/" />
+    <file src="netcoreapp1.0\build\*.targets" target="/build" />
```

Comment:
What files end up in `netcoreapp1.0\build\*.targets` that then end up in the NUPKG for this?